sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientTests.cs

```diff
@@ -3373,6 +3384,38 @@ public async Task ProcessorRenewsItsOwnershipEveryCycle()
             Assert.That(renewals, Is.EqualTo(cycles).Within(1));
         }
 
+        // <summary>
+        ///   Acts as a factory for instances of the <see cref="EventProcessorPartitionOwnership"/> class.
+        /// </summary>
+        ///
+        /// <param name="fullyQualifiedNamespace">The fully qualified Event Hubs namespace this partition ownership is associated with.  This is likely to be similar to <c>{yournamespace}.servicebus.windows.net</c>.</param>
+        /// <param name="eventHubName">The name of the specific Event Hub this partition ownership is associated with, relative to the Event Hubs namespace that contains it.</param>
+        /// <param name="consumerGroup">The name of the consumer group this partition ownership is associated with.</param>
+        /// <param name="ownerIdentifier">The identifier of the associated <c>EventProcessorClient</c> instance.</param>
+        /// <param name="partitionId">The identifier of the Event Hub partition this partition ownership is associated with.</param>
+        /// <param name="lastModifiedTime">The date and time, in UTC, that the last update was made to this ownership.</param>
+        /// <param name="version">The version needed to update this ownership.</param>
+        ///
+        /// <returns>A <see cref="PartitionOwnership" /> instance populated with the requested state</returns>
+        ///
+        private static EventProcessorPartitionOwnership CreatePartitionOwnership(string fullyQualifiedNamespace,
```

Comment:
Because it was used in two places that I thought would be harder to read with reformatting to the extended version.   Frankly, I debated doing an extension like this to avoid all the changes but figured it was better to use the type as-intended in most places.